client/reader/following-manage/index.jsx

```diff
@@ -198,7 +204,7 @@ class FollowingManage extends Component {
 							/>
 						</div> }
 				</div>
-				{ ! sitesQuery && <RecommendedSites sites={ take( recommendedSites, 2 ) } /> }
+				{ hasFollows && ! sitesQuery && <RecommendedSites sites={ take( recommendedSites, 2 ) } /> }
```

Comment:
another idea: instead of "explore discover" we pull the most recent 10 posts from discover and show them